modules/ozoneBidAdapter.js

```diff
@@ -140,14 +141,13 @@ export const spec = {
 
     if (bidderRequest && bidderRequest.gdprConsent) {
       utils.logInfo('OZONE: ADDING GDPR info');
-      ozoneRequest.regs = {};
-      ozoneRequest.regs.ext = {}; // setting default values in case there is no additional information, like for the Mirror
-      ozoneRequest.regs.ext.gdpr = bidderRequest.gdprConsent.gdprApplies ? 1 : 0;
+      let apiVersion = utils.deepAccess(bidderRequest.gdprConsent, 'apiVersion', '1');
+      ozoneRequest.regs = {ext: {gdpr: bidderRequest.gdprConsent.gdprApplies ? 1 : 0, apiVersion: apiVersion}};
       if (ozoneRequest.regs.ext.gdpr) {
         ozoneRequest.user = ozoneRequest.user || {};
         ozoneRequest.user.ext = {'consent': bidderRequest.gdprConsent.consentString};
       } else {
-        utils.logInfo('OZONE: **** Failed to find required info for GDPR for request object, even though bidderRequest.gdprConsent is TRUE ****');
+        utils.logInfo('OZONE: **** Strange CMP info: bidderRequest.gdprConsent exists BUT bidderRequest.gdprConsent.gdprApplies is false. See bidderRequest logged above. ****');
```

Comment:
Not necessarily strange behavior.

By default `gdprApplies` will be false in the scenarios where the CMP did not yet respond or errored out for any reason.

So this could in fact be a very familiar case.

Not against logging it, but gdpr can be confusing with all the different use cases and wanted to point it out.

Reply:
fair - most users who will enable logging should be aware that it maybe flagged due to the CMP not responding or timing out but will look at maybe dropping/improving this log in the future.